proposals/revised-governance.md

```diff
@@ -0,0 +1,332 @@
+share the Community Committee chair responsibilities.
+
+### This takes some power away from the CTC!
+
+Yes and no. The Core Working Group (or what we call the CTC today) would still
+have overall responsibility for nodejs/node for any aspect that is not
+specifically delegated to another working group (like Streams, for instance).
+
+For items that are under Core's responsibility, the Working Group would still
+retain the ability to make decisions following whatever process it chooses
+(likely no different than the process we have today in the CTC). However, for
+areas that have been delegated to other working groups, those working groups
+make the decisions, with any working group member having the ability to
+escalate any discussion/decision to `tsc-review`.
+
+### It will take longer to get things done!
+
+I disagree. I shouldn't take any longer than it does today.
+
+The goal is for working groups to be more empowered to make decisions on their
+own as much as possible. There will be less need to push decisions to the CTC
+and TSC and therefore less process around making decisions.
+
+### This just means we’ll have more Working Groups!
+
+I disagree. Working groups will carry with them a certain amount of required
+process. Chartering a new working group would also require consensus of all the
+other working groups, which means that there would be a fairly high bar to
+adding a new working group. This would mean that the growth of the number of
+voting seats would be much slower and manageable than the current potentially
+unbounded growth.
+
+### One monthly TSC meeting in which anyone can participate sounds like chaos!
+
+It will require better logistical planning, yes, but other groups (such as
+TC-39 for instance) pull off such meetings with very little difficulty. It
+will be the responsibility of the TSC Chair to pull together the agenda, and
+to ensure that there is adequate time for each working group to provide an
+update on its recent activity and priorities, as well as time to discuss any
+open `tsc-agenda` items.
+
+### What would happen to the existing working groups?
+
+Existing working groups would essentially be rechartered under this new process
+with a few notable changes. Specifically:
+
+* The working groups would not be chartered by the CTC (or Core Working Group).
+  Only the TSC would charter working groups.
+* Working groups would be expected to nominate one of their members to the
+  Admin team.
+* Working groups would be expected to have a bit more process around documenting
+  who their members are.
+* Several of the existing working groups would be restructured:
+  * Core Working Group (current CTC)
+  * Infrastructure and Tools Working Group
+    * Build Team
+    * Website Team
+    * Localization Teams (i18n / translation teams)
+  * Release and Support Working Group
+    * LTS Team
+    * Release Team
+    * Docker Team
+  * Streams API Working Group
+  * Intl API Working Group
```

Comment:
can this (Intl API) still be justified as a WG? is there enough activity or people involved in this work to make it something separate from core?